protoc-gen-go/generator/generator.go

```diff
@@ -484,7 +484,7 @@ func (g *Generator) CommandLineParameters(parameter string) {
 			}
 		}
 	}
-	if pluginList != "" {
+	if pluginList != "none" && pluginList != "" {
```

Comment:
Thanks for the contribution.

The generator package is deprecated. While it exists in its current, frozen state to support existing programs that may depend on it, we are no longer making or accepting changes to it.

New code should use the [protogen](https://pkg.go.dev/google.golang.org/protobuf/compiler/protogen) package instead.

Reply:
Thank you